browser/src/control/Control.NotebookbarImpress.js

```diff
@@ -1473,9 +1473,9 @@ L.Control.NotebookbarImpress = L.Control.NotebookbarWriter.extend({
 							{
 								'id': 'selectbackground',
```

Comment:
This is not related to your work but since you touched these lines, can you also re-set the id from "selectbackground" to something like "layout-selectbackground" (tab-name + function)? The function you added is id-agnostic, i think that change shouldn't be a problem.

There is an id-conflict with the "selectbackground" input item which is created inside the HTML *.m4 file. If we can also solve that, that'd be good.

Needs to be tested again of course.

Reply:
OK, I changed that and tried again and it still works.